modules/component/Makefile

```diff
@@ -20,6 +26,10 @@ IMAGE_DEPLOYED_NAME ?=
 COMPONENT_INIT_COMMAND ?= ${COMPONENT_SCRIPTS_PATH}/install-dependencies.sh
 # The command to run to execute a build
 COMPONENT_BUILD_COMMAND ?= ${COMPONENT_SCRIPTS_PATH}/build.sh
+# The command to run to execute a build the image coverage
+COMPONENT_BUILD_COVERAGE_COMMAND ?= ${COMPONENT_SCRIPTS_PATH}/build-coverage.sh
+# The command to run to execute a build the image e2e
+COMPONENT_BUILD_E2E_COMMAND ?= ${COMPONENT_SCRIPTS_PATH}/build-e2e.sh
```

Comment:
Will you be contributing example `build-coverage.sh` and `build-e2e.sh` commands to demonstrate their calling convention/parameter passing?

Reply:
where I can post examples?

Reply:
They should go right where you defined them... `component/bin`